src/constants.rs

```diff
@@ -199,6 +199,9 @@ pub const DC_LP_AUTH_NORMAL: i32 = 0x4;
 /// if none of these flags are set, the default is chosen
 pub const DC_LP_AUTH_FLAGS: i32 = DC_LP_AUTH_OAUTH2 | DC_LP_AUTH_NORMAL;
 
+/// How many existing messages shall be fetched if this feature is enabled.
```

Comment:
Is it possible to disable it? Otherwise makes sense to drop "if this feature is enabled" and say "How many existing messages shall be fetched after configuration" or something like this.